src/tests/test_dte_data_models.py

```diff
@@ -331,87 +346,95 @@ def test_ok_razon_social_none(self) -> None:
         expected_validation_errors = [
             {
                 'loc': ('receptor_razon_social',),
-                'msg': "Value must not be empty.",
+                'msg': 'Value error, Value must not be empty.',
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 self.dte_l2_1,
                 receptor_razon_social='',
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_context=False,
+            include_input=False,
+            include_url=False,
+        )
         self.assertEqual(len(validation_errors), len(expected_validation_errors))
-        for expected_validation_error in expected_validation_errors:
-            self.assertIn(expected_validation_error, validation_errors)
+        self.assertEqual(validation_errors, expected_validation_errors)
 
     def test_validate_datetime_tz(self) -> None:
         # Test TZ-awareness:
 
         expected_validation_errors = [
             {
                 'loc': ('firma_documento_dt',),
-                'msg': 'Value must be a timezone-aware datetime object.',
+                'msg': 'Value error, Value must be a timezone-aware datetime object.',
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 self.dte_l2_1,
                 firma_documento_dt=datetime(2019, 4, 5, 12, 57, 32),
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_context=False,
+            include_input=False,
+            include_url=False,
+        )
         self.assertEqual(len(validation_errors), len(expected_validation_errors))
-        for expected_validation_error in expected_validation_errors:
-            self.assertIn(expected_validation_error, validation_errors)
+        self.assertEqual(validation_errors, expected_validation_errors)
 
         # Test TZ-value:
 
         expected_validation_errors = [
             {
                 'loc': ('firma_documento_dt',),
                 'msg': (
-                    '('
-                    '''"Timezone of datetime value must be 'America/Santiago'.",'''
-                    ' datetime.datetime(2019, 4, 5, 12, 57, 32, tzinfo=<UTC>)'
-                    ')'
+                    'Value error, ('
+                    '''"Timezone of datetime value must be 'America/Santiago'.", '''
+                    'datetime.datetime(2019, 4, 5, 12, 57, 32, tzinfo=<UTC>))'
```

Comment:
Reformat to reduce diff size.

```suggestion
                    '''"Timezone of datetime value must be 'America/Santiago'.",'''
                    ' datetime.datetime(2019, 4, 5, 12, 57, tzinfo=<UTC>)'
                    ')'
```